modules/guardduty/variables.tf

```diff
@@ -0,0 +1,16 @@
+# Assume special IAM-Role (../iam-role) in remote account to create roles and policies
+provider "aws" {
+  region = "${var.region}"
+
+  assume_role {
+    role_arn = "arn:aws:iam::${var.assume_role_in_account_id}:role/${var.landing_iam_role}"
```

Comment:
Almost mate.  You just need to create the two variables referenced here.  @sh31 should be able to show you what they're doing. 🙏